code/game/objects/structures/displaycase.dm

```diff
@@ -191,6 +191,9 @@
 		update_icon()
 		return
 	else
+	    //prevents remote "kicks" with TK
+		if (abs(usr.x-src.x) > 1 || abs(usr.y-src.y) > 1)

```

Comment:
user.adjacent(src) instead 

in general we avoid usr where a proper user variable is available